### ISSUE TITLE
Donate page accessibility issues

### ISSUE THREAD

**Issue by gixxerblade** [Vets-Who-Code/vets-who-code-app#225]
# Accessibility

## `<frame>` or `<iframe>` elements do not have a title [Learn More](https://web.dev/frame-title/?utm_source=lighthouse&utm_medium=devtools)

Users of screen readers and other assistive technologies rely on frame titles to describe the contents of frames. Navigating through frames and inline frames can quickly become difficult and confusing for assistive technology users if the frames are not marked with a title attribute.

### Failing Elements

- `<iframe src="https://donorbox.org/embed/vetswhocode-donation?show_content=true" seamless="" name="donorbox" scrolling="yes" allowpaymentrequest="true" frameborder="0" tabindex="0"></iframe>`

## Heading elements are not in a sequentially-descending order 

Screen readers have commands to quickly jump between headings or to specific landmark regions.

### Failing Elements

`<h4 class="cause_title">Help Us Continue To Train More Veterans</h4>`

**To Reproduce**
1. Go to 'https://vetswhocode.io/donate'
2. Click on 'Developer Tools (F12)'
3. Click on Lighthouse
4. Generate report

**Expected behavior**
- `<iframe>` elements do not have a title - Add 'aria-label` to `iframe` element
- Heading elements are not in a sequentially-descending order - The element should be ordered correctly based on the current markup.

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by gixxerblade** [comment]
closed with #227
